roles/container-host/tasks/container_mirror.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: "Create {{ container_mirror_cert_path }}"
+  file:
+    path: "{{ container_mirror_cert_path }}"
+    state: directory
+
+- name: "Copy {{ container_mirror }} self-signed cert"
+  copy:
+    dest: "{{ container_mirror_cert_path }}/docker-mirror.crt"
+    content: "{{ container_mirror_cert }}"
+
+- name: Install registries-conf-ctl 
+  pip:
+    name: git+https://github.com/sebastian-philipp/registries-conf-ctl
+    state: latest
+
```

Comment:
executable: pip3.3

Maybe this could avoid the search of the installed package ... just an idea

Reply:
We don't have pip3 on all our testnodes.